build/protobuf.mk

```diff
@@ -44,13 +44,21 @@ CPP_PROTOS := $(filter %api.proto %data.proto %internal.proto %config.proto %err
 CPP_HEADERS := $(subst ./,$(ENGINE_ROOT)/,$(CPP_PROTOS:%.proto=%.pb.h)) $(subst $(GOGOPROTO_ROOT),$(ENGINE_ROOT),$(GOGOPROTO_PROTO:%.proto=%.pb.h))
 CPP_SOURCES := $(subst ./,$(ENGINE_ROOT)/,$(CPP_PROTOS:%.proto=%.pb.cc)) $(subst $(GOGOPROTO_ROOT),$(ENGINE_ROOT),$(GOGOPROTO_PROTO:%.proto=%.pb.cc))
 
+ENGINE_PROTOS := $(sort $(shell find $(ENGINE_ROOT) -name *.proto -type f))
```

Comment:
The protos aren't c++ specific but the way this variable is used is to generate c++ code. These same protos also appear in GO_PROTOS and the go code is generated from their inclusion in that variable, not ENGINE_PROTOS.